### PR TITLE
[#19] WIP - CircleCI initial test config

### PR DESCRIPTION
**Pull request by LucaPrete**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      command: |
+        apk add ca-certificates git openssh gcc libc-dev
+
+  get_dependencies: &get_dependencies
+    run:
+      name: get-dependencies
+      command: |
+        go get github.com/teamdigitale/anpr-dashboard-server/server
+        go get github.com/teamdigitale/anpr-dashboard-server/converter
+        go get github.com/stretchr/testify/assert
```
Comment (sebbalex) [inline .circleci/config.yml:24]: I would replace it with a more comprehensive `go get -d ./...`

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  test: &test
+    run:
+      name: test
+      command: |
+        cd server && go test -race
```
Comment (sebbalex) [inline .circleci/config.yml:30]: I found out that some other tests lived in other folder. In order to get them all I would avoid `cd server` and run test with all flag as `go test -race ./...`

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (LucaPrete) [reply .circleci/config.yml:24]: Unfortunately, this starts to recursively download a bunch of dependencies from all over and the operation lasts "forever". I'd suspect it's not only downloading the dependencies needed

Comment (sebbalex) [reply .circleci/config.yml:24]: we will investigate in a separate issue.

**Review by sebbalex — APPROVED** [review]
👍

File: .circleci/config.yml (hunk 2 above)
Comment (sebbalex) [reply .circleci/config.yml:30]: fixed with go 1.13

**Pull request merged**